apps/openmw/mwphysics/actor.cpp

```diff
@@ -28,39 +26,46 @@ Actor::Actor(const MWWorld::Ptr& ptr, osg::ref_ptr<const Resource::BulletShape>
     mHalfExtents = shape->mCollisionBoxHalfExtents;
     mMeshTranslation = shape->mCollisionBoxTranslate;
 
-    // Use capsule shape only if base is square (nonuniform scaling apparently doesn't work on it)
-    if (std::abs(mHalfExtents.x()-mHalfExtents.y())<mHalfExtents.x()*0.05 && mHalfExtents.z() >= mHalfExtents.x())
+    // the ratio between the side length and inradius of a regular octagon is 2tan(pi/8), which is equivalent to 2(sqrt(2)-1)
```

Comment:
The following value is equivalent to tan(pi/8) or sqrt(2)-1. Is the comment or the constant incorrect?

Reply:
The variable is half the actual edge-inradius ratio, because it's measured from the axis, not the other vertex of the edge.

Reply:
I can elaborate on the variable's comment if you want.

Reply:
Makes sense.